cogs/assessments.py

```diff
@@ -0,0 +1,103 @@
+import json
+import os
+import re
+import arrow
+import discord
+import requests
+from discord.ext import commands
+import time
+import sys
+from datetime import datetime
+
+assessments = []
+
+
+class Assessments(commands.Cog):
+    def __init__(self,bot):
+        self.bot = bot
+        self.load_assessments_file()
+
+    def load_assessments_file(self):
+        #Load file which contains all upcoming assessments
+        if os.path.isfile("assessments.txt"):
```

Comment:
"assessments.txt" is referenced multiple times, so it would be best to have it as a class variable.